app/routes/companyInterest/CompanyInterestRoute.ts

```diff
@@ -58,7 +58,7 @@ const mapStateToProps = (state, props) => {
         checked: false,
       })),
       participantRange: null,
-      companyTypes: null,
+      companyType: null,
```

Comment:
This was misspelled, but had no affect on anything. Not sure why it is even needed.